public/content/developers/docs/data-and-analytics/index.md

```diff
@@ -1,55 +1,122 @@
 ---
-title: Data and analytics
-description: How to get on-chain analytics and data for use in your dapps
+title: Data and Analytics
+description: Understanding and navigating the crypto data stack
 lang: en
 ---
 
 ## Introduction {#Introduction}
 
-As utilization of the network continues to grow, an increasing amount of valuable information will exist in the on-chain data. As the volume of data rapidly increases, calculating and aggregating this information to report upon or drive a dapp can become a time and process heavy endeavor.
+As you get into crypto, you're likely to see stats, charts, and APIs across dozens of different websites (and github repos). This is a stark contrast from traditional tech or finance where there is just one trusted source (i.e. Bloomberg). This is due to the open data nature of the industry, where all data can be freely downloaded from client nodes.
 
-Leveraging existing data providers can expedite development, produce more accurate results, and reduce ongoing maintenance efforts. This will enable a team to concentrate on the core functionality their project is trying to provide.
+However, that data is stored in hex/binary form and is hardly legible or usable. As an ecosystem, we've had to work together to share queries, tools, tables, and labor to make crypto data accessible for everyone. Each year, we see giant leaps in how we can combine onchain/offchain data in analytics and applications. 
 
-## Prerequisites {#prerequisites}
+In this overview page, we'll cover all the parts of the crypto data stack as they stand today (last updated in 2024).
```

Comment:
@andrewhong5297 Certainly apologize for delays here... anything critical that has developed since that you'd like to add now that we're in 2025?